.lints/rules/deny-import-serde-json.yml

```diff
@@ -17,7 +17,7 @@ severity: error
 language: Rust
 message: "ambiguous import from `serde_json` crate"
 note: |
-  reason: this import may lead to ambiguity between items imported from `variant` and `serde_json` crates.
```

Comment:
IIRC we don't use jsonbb in this crate. Perhaps we can drop this rule?

Reply:
`Value` is vague IMO, maybe I could improve the note.